### PR TITLE
Add introText property to PackageChange

### PR DESCRIPTION
**Pull request by meel-io**
### Description
Added optional prop `introText` to PackageChange. If present, it'll render it as secondary text below the package name.

### Ticket
https://financialtimes.atlassian.net/browse/UG-431

### Screenshots

| Before | After |
| ------ | ----- |
|before|after|

### Reminder
Have you completed these common tasks (remove those that don't apply)?

- [x] **Tests** written for new or updated for existing functionality
- [x] **Stories** updated to use this change
- [x] **Product Review** ran past the product owner

### CODE REVIEW

File: components/package-change.jsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export function PackageChange({ changePackageUrl, currentPackage }) {
+export function PackageChange({ changePackageUrl, currentPackage, introText }) {
```
Comment (meel-io) [inline components/package-change.jsx:4]: I'm not so sure about the name `introText`. Any better ideas? 💭

**Review by ststdimov — APPROVED** [review]

**Review by maguero — DISMISSED** [review]

File: components/package-change.stories.js [hunk 2]
```diff
@@ -10,4 +10,5 @@ export const Basic = (args) => <PackageChange {...args} />;
 Basic.args = {
 	currentPackage: 'Premium Digital',
 	changePackageUrl: 'https://ft.com/products',
+	introText: 'Personalised email briefings and alerts',
```
Comment (ekocian) [inline components/package-change.stories.js:13]: **suggestion**: If the new field is not required, I would create a separate story

**Review by ekocian — APPROVED** [review]
LGTM! minnor comment

File: components/package-change.jsx (hunk 1 above)
Comment (jamesnicholls) [reply components/package-change.jsx:4]: `packageDescription`?

Comment (meel-io) [reply components/package-change.jsx:4]: Sounds better ✅

File: components/package-change.jsx [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 		<div className="ncf__package-change">
 			<div className="ncf__package-change__package">
-				<p className="ncf__package-change__content">
-					You have chosen <span className="ncf__strong">{currentPackage}</span>
-				</p>
+				<div className="ncf__package-change__content">
+					<p>
+						You have chosen <span className="ncf__strong">{currentPackage}</span>
+					</p>
+					{packageDescription && <p className="ncf__package-change__content__description">{packageDescription}</p>}
```
Comment (jamesnicholls) [inline components/package-change.jsx:12]: **thought (non-blocking):** The use of a nested element here (`__content__description`) is a bit of a code smell. It couples the class name to the structure of the block (the description always needs to a be child of the content block) and would make it difficult to change the structure in the future.

That said, the whole `n-conversion-forms` library seems to be using nested elements (e.g. `ncf__package-change__content`). I think the `ncf__` prefix here is being used as a namespace and should probably be changed to `ncf-`. That is outside of the scope of this PR though.

**Review by jamesnicholls — APPROVED** [review]
This looks good to me. I have added a note about the use of nested BEM elements, but looking at other classes used in this library, it would require a big change outside the scope of this PR.

**Review by ststdimov — APPROVED** [review]

File: components/package-change.jsx (hunk 3 above)
Comment (meel-io) [reply components/package-change.jsx:12]: I see what you mean. I can add it as Tech Debt and see if it's worth tackling sometime in the future.

**Pull request merged**